<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pview</class>
 <widget class="QWidget" name="pview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>pview</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="ShotBox">
     <property name="title">
      <string>ShotBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QWidget" name="shotImageWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <layout class="QGridLayout" name="shotImageLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_9">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_12">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>1</number>
              </property>
              <property name="leftMargin">
               <number>1</number>
              </property>
              <property name="topMargin">
               <number>1</number>
              </property>
              <property name="rightMargin">
               <number>1</number>
              </property>
              <property name="bottomMargin">
               <number>1</number>
              </property>
              <item>
               <widget class="QGroupBox" name="groupBox_14">
                <property name="maximumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="title">
                 <string>Status</string>
                </property>
                <layout class="QGridLayout" name="gridLayout_22">
                 <property name="leftMargin">
                  <number>1</number>
                 </property>
                 <property name="topMargin">
                  <number>1</number>
                 </property>
                 <property name="rightMargin">
                  <number>1</number>
                 </property>
                 <property name="bottomMargin">
                  <number>1</number>
                 </property>
                 <property name="spacing">
                  <number>1</number>
                 </property>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="shotStatus"/>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <widget class="QFrame" name="frame_3">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QGridLayout" name="gridLayout_23">
                 <property name="leftMargin">
                  <number>1</number>
                 </property>
                 <property name="topMargin">
                  <number>1</number>
                 </property>
                 <property name="rightMargin">
                  <number>1</number>
                 </property>
                 <property name="bottomMargin">
                  <number>1</number>
                 </property>
                 <property name="spacing">
                  <number>1</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QGroupBox" name="groupBox_6">
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>70</height>
                    </size>
                   </property>
                   <property name="title">
                    <string>Frame Range</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_16">
                    <property name="leftMargin">
                     <number>1</number>
                    </property>
                    <property name="topMargin">
                     <number>1</number>
                    </property>
                    <property name="rightMargin">
                     <number>1</number>
                    </property>
                    <property name="bottomMargin">
                     <number>1</number>
                    </property>
                    <property name="spacing">
                     <number>1</number>
                    </property>
                    <item row="0" column="0">
                     <widget class="QSpinBox" name="startFrame">
                      <property name="maximum">
                       <number>999999</number>
                      </property>
                      <property name="value">
                       <number>101</number>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QSpinBox" name="endFrame">
                      <property name="minimum">
                       <number>1</number>
                      </property>
                      <property name="maximum">
                       <number>999999</number>
                      </property>
                      <property name="value">
                       <number>201</number>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>-</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="shotDescriptionGroupBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Shot Description</string>
        </property>
        <layout class="QGridLayout" name="shotDescriptionLayout">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <property name="spacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Shot Notes</string>
         </attribute>
         <layout class="QGridLayout" name="shotNotesLayout">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tasks</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_17">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>252</width>
               <height>229</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout_25">
              <item row="0" column="0">
               <widget class="QTreeWidget" name="taskTable">
                <property name="dragEnabled">
                 <bool>true</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::InternalMove</enum>
                </property>
                <property name="defaultDropAction">
                 <enum>Qt::TargetMoveAction</enum>
                </property>
                <property name="alternatingRowColors">
                 <bool>true</bool>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
                <property name="verticalScrollMode">
                 <enum>QAbstractItemView::ScrollPerItem</enum>
                </property>
                <property name="itemsExpandable">
                 <bool>true</bool>
                </property>
                <property name="sortingEnabled">
                 <bool>false</bool>
                </property>
                <property name="expandsOnDoubleClick">
                 <bool>true</bool>
                </property>
                <attribute name="headerHighlightSections">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerMinimumSectionSize">
                 <number>100</number>
                </attribute>
                <attribute name="headerShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="headerStretchLastSection">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Type</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Artist</string>
                 </property>
                </column>
                <column>
                 <property name="text">
                  <string>Done</string>
                 </property>
                </column>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
